starter-code/app.js

```diff
@@ -8,9 +8,9 @@ const expressLayouts = require('express-ejs-layouts');
 const mongoose       = require('mongoose');
 const index          = require('./routes/index');
 const users          = require('./routes/users');
-const drones         = require('./routes/drones');
+const dronesRouter        = require('./routes/drones.route');
```

Comment:
Perfecto! Ya que no usáis drones.js en routes, podríais haberlo borrado que ya no os sirve de nada.